src/neuroconv/datainterfaces/ophys/brukertiff/brukertiffdatainterface.py

```diff
@@ -0,0 +1,81 @@
+from pathlib import Path
+from typing import Optional
+from xml.etree import ElementTree
+
+import numpy as np
+from dateutil.parser import parse
+
+from ..baseimagingextractorinterface import BaseImagingExtractorInterface
+from ....utils import FolderPathType
+
+
+class BrukerTiffImagingInterface(BaseImagingExtractorInterface):
+    """Data Interface for BrukerTiffImagingExtractor."""
+
+    @classmethod
+    def get_source_schema(cls) -> dict:
+        source_schema = super().get_source_schema()
+        source_schema["properties"]["folder_path"]["description"] = "Path to Tiff folder."
```

Comment:
Can you be a little more precise here? Do you mean the path to the folder that contains tiff files?